File: servico-musica-java/java.graphql/src/main/java/servico/java/graphql/controller/UsuarioController.java
package servico.java.graphql.controller;

import org.springframework.graphql.data.method.annotation.QueryMapping;
import org.springframework.stereotype.Controller;
import servico.java.graphql.entity.Usuario;
import servico.java.graphql.repository.UsuarioRepository;

import java.util.List;

@Controller
public class UsuarioController {

    private final UsuarioRepository usuarioRepository;

    public UsuarioController(UsuarioRepository usuarioRepository) {
        this.usuarioRepository = usuarioRepository;
    }

    @QueryMapping
    public List<Usuario> todosUsuarios() {
        return usuarioRepository.findAll();
    }
}

File: servico-musica-kotlin/kotlin.graphql/src/main/kotlin/servico/kotlin/graphql/controller/UsuarioController.kt
package servico.kotlin.graphql.controller

import org.springframework.graphql.data.method.annotation.QueryMapping
import org.springframework.stereotype.Controller
import servico.kotlin.graphql.entity.Usuario
import servico.kotlin.graphql.repository.UsuarioRepository

@Controller
class UsuarioController(private val usuarioRepository: UsuarioRepository) {
    @QueryMapping
    fun todosUsuarios(): MutableList<Usuario> {
        return usuarioRepository.findAll()
    }
}